packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -434,7 +434,9 @@ class Metrics {
       ...metricTags,
       browser: bowser.name,
       org_id: this.webex.credentials.getOrgId(),
-      os: bowser.osname
+      os: bowser.osname,
+      domain: window.location.hostname,
+      clientid: this.webex.credentials.config._values.client_id
```

Comment:
Can you keep it consistent with other measures and update the key?
```suggestion
      client_id: this.webex.credentials.config._values.client_id
```